mosaic2/include/mosaic2.inc

```diff
@@ -86,15 +86,13 @@
   !> <br>Example usage:
   !!            call calc_mosaic_grid_area(lon, lat, area)
   subroutine CALC_MOSAIC_GRID_AREA_(lon, lat, area)
-     real(kind=FMS_MOS_KIND_), dimension(:,:), intent(in)    :: lon
-     real(kind=FMS_MOS_KIND_), dimension(:,:), intent(in)    :: lat
-     real(kind=FMS_MOS_KIND_), dimension(:,:), intent(inout) :: area
```

Comment:
I am asking purely for my own learning and do not know what is best: Why would you decide to change the intent instead of just initializing the output variable in the code before this subroutine is called?

Reply:
`CALC_MOSAIC_GRID_AREA_` is a wrapper around `get_grid_area`, which sets the area but doesn't read the value that's passed in. So `intent(out)` more clearly communicates to the user what the code is doing.

Reply:
Or in other words, there's no reason it should need to be initialized, because it gets set by `get_grid_area`.